src/components/atoms/Image.js

```diff
@@ -0,0 +1,4 @@
+import React from 'react';
+export default function Image({source}){
+    return <img src={source} alt="" />
```

Comment:
image must accept props for URL & alt

alt is very important from the Accessibility point of view

try using prop-types and defining prop types